docs/embedded/concepts/app-concepts/auth.md

```diff
@@ -24,6 +24,58 @@ SharePoint Embedded supports both App-Only and Delegated (App+User) calls for en
 
 Both App-Only and Delegated SharePoint Embedded from trusted (or private) client applications are allowed. SharePoint Embedded calls to create containers, either App-Only or Delegated, from public clients are blocked by SharePoint Embedded authorization management.
 
+## Container.Selected Scope 
+
+The [Register Container Type API](register-api-documentation.md) API uses the Container.Selected scope. You will to call this API. To use it, you must configure this scope in your App manifest.  
+
+>>  Note that other SharePoint Embedded Graph APIs run with the FileStorageContainer.Selected Scope on Microsoft Graph 
```

Comment:
- this should use the MSDOCS alert format
- `Scope` should be lowercase
- should the scope be bold?